<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>500</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>700</width>
    <height>500</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>700</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>700</width>
      <height>461</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="page_menu">
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>50</y>
        <width>140</width>
        <height>40</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:14pt; font-weight:600; text-decoration: underline;&quot;&gt;MFS&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:9pt;&quot;&gt;  (Music File System)&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>110</y>
        <width>130</width>
        <height>40</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:11pt; font-weight:600;&quot;&gt;Menu Principal&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_ir_a_crear">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>320</y>
        <width>100</width>
        <height>26</height>
       </rect>
      </property>
      <property name="text">
       <string>Crear un Disco!</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_ir_a_abrir">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>320</y>
        <width>130</width>
        <height>26</height>
       </rect>
      </property>
      <property name="text">
       <string>Abrir un Disco!</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_crear">
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>30</y>
        <width>100</width>
        <height>15</height>
       </rect>
      </property>
      <property name="text">
       <string>Crear disco</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>260</y>
        <width>150</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Nombre y dir. del disco:  </string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>150</y>
        <width>120</width>
        <height>15</height>
       </rect>
      </property>
      <property name="text">
       <string>No. de File Entries: </string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>200</y>
        <width>130</width>
        <height>15</height>
       </rect>
      </property>
      <property name="text">
       <string>No. de Tag Entries: </string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_crear">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>360</y>
        <width>80</width>
        <height>26</height>
       </rect>
      </property>
      <property name="text">
       <string>Create it!</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_back">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>370</y>
        <width>30</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;--</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_bloques">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>90</y>
        <width>113</width>
        <height>25</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>90</y>
        <width>130</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Cantidad de bloques</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_FE">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>140</y>
        <width>113</width>
        <height>25</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_TE">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>200</y>
        <width>113</width>
        <height>25</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_disco">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>260</y>
        <width>113</width>
        <height>25</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_getDir">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>260</y>
        <width>30</width>
        <height>26</height>
       </rect>
      </property>
      <property name="text">
       <string>...</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_arbir">
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>110</y>
        <width>210</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>Escriba la direccion de su disco: </string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_disco2">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>110</y>
        <width>113</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>disco.dat</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_getopen">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>110</y>
        <width>31</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>...</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_abrir">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>210</y>
        <width>80</width>
        <height>26</height>
       </rect>
      </property>
      <property name="text">
       <string>Explorar</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_back2">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>310</y>
        <width>30</width>
        <height>26</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;--</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_explorar">
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>10</y>
        <width>400</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>DIsco </string>
      </property>
     </widget>
     <widget class="QListWidget" name="listWidget_tags">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>60</y>
        <width>310</width>
        <height>250</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>TAGS</string>
       </property>
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
         <underline>true</underline>
        </font>
       </property>
       <property name="textAlignment">
        <set>AlignHCenter|AlignVCenter|AlignCenter</set>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Show All Songs...</string>
       </property>
       <property name="font">
        <font>
         <italic>true</italic>
        </font>
       </property>
      </item>
     </widget>
     <widget class="QListWidget" name="listWidget_songs">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>60</y>
        <width>310</width>
        <height>250</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>SONGS</string>
       </property>
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
         <underline>true</underline>
        </font>
       </property>
       <property name="textAlignment">
        <set>AlignHCenter|AlignVCenter|AlignCenter</set>
       </property>
      </item>
     </widget>
     <widget class="Line" name="line">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>60</y>
        <width>3</width>
        <height>250</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_addSong">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>340</y>
        <width>110</width>
        <height>26</height>
       </rect>
      </property>
      <property name="text">
       <string>Agregar Cancion</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_export">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>340</y>
        <width>110</width>
        <height>26</height>
       </rect>
      </property>
      <property name="text">
       <string>Exportar</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_delete">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>340</y>
        <width>110</width>
        <height>26</height>
       </rect>
      </property>
      <property name="text">
       <string>Eliminar</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_tag">
      <property name="geometry">
       <rect>
        <x>560</x>
        <y>340</y>
        <width>110</width>
        <height>26</height>
       </rect>
      </property>
      <property name="text">
       <string>Agregar TAG</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_next_page">
      <property name="geometry">
       <rect>
        <x>630</x>
        <y>430</y>
        <width>40</width>
        <height>26</height>
       </rect>
      </property>
      <property name="text">
       <string>--&gt;</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_goback3">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>430</y>
        <width>40</width>
        <height>26</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;--</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_mp3">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>390</y>
        <width>230</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_tag">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>430</y>
        <width>200</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>390</y>
        <width>30</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>Mp3:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_13">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>430</y>
        <width>30</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>Tag:</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_searSong">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>390</y>
        <width>30</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>...</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_14">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>390</y>
        <width>140</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>Nombre del mp3 a exportar: </string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_mp3_exportado">
      <property name="geometry">
       <rect>
        <x>520</x>
        <y>390</y>
        <width>150</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_comandos">
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>20</y>
        <width>100</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>DISCO TAL</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>110</y>
        <width>80</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Comando:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_comando">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>110</y>
        <width>290</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QTextEdit" name="textEdit_resultados">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>150</y>
        <width>380</width>
        <height>210</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_previous_page">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>390</y>
        <width>40</width>
        <height>26</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;--</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_exec">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>110</y>
        <width>60</width>
        <height>26</height>
       </rect>
      </property>
      <property name="text">
       <string>Ejecutar</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_textEdit">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>180</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Limpiar</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>700</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
